qutebrowser/browser/downloads.py

```diff
@@ -492,6 +492,29 @@ def __str__(self):
                         remaining=remaining, perc=perc, down=down,
                         total=total, errmsg=errmsg))
 
+    def _find_next_filename(self) -> None:
+        """Append unique numeric suffix to filename.
+
+        After finding a unique filename, restart the process of setting the
+        filename which will update the filename shown in the downloads list.
+        """
+        assert self._filename is not None
+        path, file = os.path.split(self._filename)
+        match = re.fullmatch(r'(.+?)((\.[a-z]+)?\.[^.]+)', file)
+        if match:
+            base, suffix = match[1], match[2]
+        else:
+            base = file
+            suffix = ''
+
+        for i in range(2, 1000):
```

Comment:
Should this start at `1`? idk, I feel like I remember "(1)"s in the past (on windows?)